lib/src/space/mod.rs

```diff
@@ -61,7 +61,7 @@ pub trait SpaceObserver {
 
 /// A reference to a SpaceObserver that has been registered with a Space
 #[derive(Clone)]
-pub struct SpaceObserverRef<T: SpaceObserver> (Rc<RefCell<T>>);
+pub struct SpaceObserverRef<T: SpaceObserver> (pub Rc<RefCell<T>>);
```

Comment:
Could we introduce additional functions to hide internal representation instead of making it public? Otherwise API users can make dependency on the behaviour which we may like to change in future.

Reply:
Done in ec321fb